src/calendar.h

```diff
@@ -248,7 +248,7 @@ class calendar
         /** Returns the translated name of the season (with first letter being uppercase). */
         static const std::string name_season( season_type s );
 
-        static   calendar start;
+        static   calendar start_of_cataclysm;
```

Comment:
And this is actually the storage for when the game started (e.g. next summer if you choose that specific scenario) and has nothing to do with the cataclysm at all. The cataclysm happened at `calendar::time_of_cataclysm`.

Reply:
Not so, the previous names were basically reversed. From @esotericist in #31953:
>right now `calendar::time_of_cataclysm` is essentially the zero-point of the game's simulation, and not actually related to when the cataclysm itself occurred. That honor goes to `calendar::start`, which is also not super clear by name alone.

Next summer scenario must set the spawn delay instead, which is start delay *after* the cataclysm occurs.

Reply:
Please take a look at `game::start_calendar`. It sets `calendar::start` depending on various factors. It has nothing to do with when the cataclysm happens.

> calendar::time_of_cataclysm is essentially the zero-point of the game's simulation

Yes, and that zero part is an implementation detail and should not matter to any code and therefor it should not be exposed.

Reply:
It's set to the value of "initial day", aka the day of the cataclysm, unless the scenario chooses a season for the cataclysm to occur in.

Agreed on the zero part being technically irrelevant, but apparently eso and Kevin agreed on these names, so who am I to argue?